blockchain/blockchain.go

```diff
@@ -159,6 +159,18 @@ type BlockChain struct {
 
 	nonceCache   common.Cache
 	balanceCache common.Cache
+
+	// State migration
+	prepareStateMigration    bool
+	chStateTrieMigrationTask chan stateTrieMigrationTask
+	stopMigration            chan struct{}
+	committedCnt             int
+	pendingCnt               int
```

Comment:
These variables may be moved to `stateTrieMigrationTask`.

Reply:
I removed `stateTrieMigrationTask` with the loop.